hail_scripts/utils/clinvar.py

```diff
@@ -21,9 +40,68 @@
         "practice_guideline": 4,
     }
 )
+CLINVAR_PATHOGENICITIES_LOOKUP = hl.dict(
+    hl.enumerate(
+        # NB: sorted by pathogenicity
+        [
+            'Pathogenic',
+            'Pathogenic/Likely_pathogenic',
+            'Pathogenic/Likely_pathogenic/Likely_risk_allele',
+            'Pathogenic/Likely_risk_allele',
+            'Likely_pathogenic',
+            'Likely_pathogenic/Likely_risk_allele',
+            'Likely_risk_allele',
+            'Conflicting_interpretations_of_pathogenicity',
+            'Established_risk_allele',
+            'Uncertain_risk_allele',
+            'Uncertain_significance/Uncertain_risk_allele',
+            'Uncertain_significance',
+            'Likely_benign',
+            'Benign/Likely_benign',
+            'Benign',
+            CLINVAR_DEFAULT_PATHOGENICITY,
+        ],
+        index_first=False
+    )
+)
 
-
-def download_and_import_latest_clinvar_vcf(genome_version: str) -> hl.MatrixTable:
+def parsed_clnsig(ht: hl.Table):
+    return (
+        hl.delimit(ht.info.CLNSIG)
+        .replace(
+            'Likely_pathogenic,_low_penetrance', 'Likely_pathogenic|low_penetrance',
+        )
+        .replace(
+            '/Pathogenic,_low_penetrance', '|low_penetrance',
+        )
+        .split(r'\|')
+    )
+
+def parsed_clnsigconf(ht: hl.Table):
+
+    def parse_to_count(entry: str):
```

Comment:
is there a reason to have this be a nested function?

Reply:
Reasoning for it being nested is it's just `lambda` I moved up so that I could assign a variable.  It shouldn't have a meaning outside of this function.  Happy to move it up though!

Reply:
Yeah I think moving it up would be clearer